signing_service/signing_service/tests/test_authenticate.py

```diff
@@ -125,6 +126,7 @@ def mocked_generator():
                         SIGNING_SERVICE_PRIVATE_KEY,
                         TEST_ETHEREUM_PRIVATE_KEY,
                         SIGNING_SERVICE_DEFAULT_RECONNECT_ATTEMPTS,
+                        Notifier(None, None, ['']),
```

Comment:
In tests, `SigningService` should be fed with `ConsoleNotifier` instance

Reply:
Done